contracts/wrapperContracts/WrapFeeBurner.sol

```diff
@@ -0,0 +1,227 @@
+pragma solidity 0.4.18;
+
+
+import "../ERC20Interface.sol";
+import "../FeeBurner.sol";
+import "./WrapperBase.sol";
+
+
+contract WrapFeeBurner is WrapperBase {
+
+    FeeBurner public feeBurnerContract;
+    address[] internal feeSharingWallets;
+    uint public feeSharingBps = 3000;
+
+    //knc rate range
+    struct KncPerEth {
+        uint minRate;
+        uint maxRate;
+        uint pendingMinRate;
+        uint pendingMaxRate;
+    }
+
+    KncPerEth private kncPerEth;
+
+    //add reserve pending data
+    struct AddReserveData {
+        address reserve;
+        uint    feeBps;
+        address kncWallet;
+    }
+
+    AddReserveData private addReserve;
+
+    //wallet fee parameters
```

Comment:
consider writing "wallet fee **pending** parameters", to align with other comments.
This will be clearer since you don't used the "pending" prefix in the variable names.